SIMFractureQstatic.h

```diff
@@ -131,6 +132,18 @@ class SIMFractureQstatic : public SIMFracture<SolidSlv,PhaseSlv,SIMCoupledSI>
     return SIM::DIVERGED;
   }
 
+  //! \brief Returns the residual of the elastic equation.
+  virtual const Vector& getAitkenResidual() const
+  { return this->disResidual; }
+
+  //! \brief Returns solution to use for relaxation.
+  virtual const Vector& getRelaxationVector() const
+  { return this->S2.getSolution(); }
+
+  //! \brief Set the relaxed solution.
+  virtual void setRelaxedSolution(const Vector& sol)
+  { this->S2.setSolution(sol); }
+
```

Comment:
Maybe put these methods in parent class instead? Since `calcResidual()` - the only user - is placed there.

Reply:
well, it would work since we do not use override in these parts, but strictly speaking SIMFractureDynamics can be instanced for SIMCoupled in which case there are not virtual methods to override. I can do it, but I consider it slightly wrong.

Reply:
(Also calcResidual is not the user, SIMCoupledSI::solveStep is now the user)